lib/programs/exchange_format.rb

```diff
@@ -1,4 +1,4 @@
-module ExchangeFormat
+module Programs::ExchangeFormat
```

Comment:
Since `lib/` is now a top level namespace, zeitwerk is expecting `programs/` to define a new namespace.  All programs now reside in the namespace, and allows us to easily get all constants of this namespace in `run.rb` instead of needing to manually name all programs there every time we change something.

Reply:
👍 Love it!